### PR TITLE
Bump CRDS context to 513 for tests

### PR DESCRIPTION
**Pull request by hbushouse**
Increment context again. 513 is the current latest delivery.

### CODE REVIEW

**Comment by hbushouse** [comment]
FYI, 508 checked out OK. Only changes in regression tests were 3 tests affected by a new MIRI readnoise ref file.

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ env:
   global:
     - CRDS_SERVER_URL='https://jwst-crds.stsci.edu'
     - CRDS_PATH='/tmp/crds_cache'
-    - CRDS_CONTEXT="jwst_0508.pmap"
+    - CRDS_CONTEXT="jwst_0513.pmap"
```
Comment (jdavies-st) [inline .travis.yml]: This line can be removed entirely.  Normally we do not set the CRDS_CONTEXT for our CI unit tests.  We just use the default.

File: Jenkinsfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ python_version = '3.6'
 env_vars = [
     "CRDS_SERVER_URL=https://jwst-crds.stsci.edu",
     "CRDS_PATH=./crds_cache",
-    "CRDS_CONTEXT=jwst_0508.pmap",
+    "CRDS_CONTEXT=jwst_0513.pmap",
```
Comment (jdavies-st) [inline Jenkinsfile]: Same here.  This line can be deleted so that we just use the default.

File: JenkinsfileRT [hunk 3]
```diff
@@ -13,7 +13,7 @@ numpy_version = '1.16'
 env_vars = [
     "TEST_BIGDATA=https://bytesalad.stsci.edu/artifactory",
     "CRDS_SERVER_URL=https://jwst-crds.stsci.edu",
-    "CRDS_CONTEXT=jwst_0508.pmap",
+    "CRDS_CONTEXT=jwst_0513.pmap",
```
Comment (jdavies-st) [inline JenkinsfileRT]: And this can be set to `jwst-edit`

Comment (jdavies-st) [reply JenkinsfileRT]: I.e. revert this:

https://github.com/spacetelescope/jwst/commit/78fc038294c4e5a2ca5e507f04ca2cb6797ab31a#diff-d989dbe11b6a131293f54f52a6458757

File: .travis.yml (hunk 1 above)
Comment (hbushouse) [reply .travis.yml]: Done

File: Jenkinsfile (hunk 2 above)
Comment (hbushouse) [reply Jenkinsfile]: Done

File: JenkinsfileRT (hunk 3 above)
Comment (hbushouse) [reply JenkinsfileRT]: Done

**Comment by hbushouse** [comment]
Jenkins is hosed again due to "pip install -e" thing. Merging.

**Comment by jdavies-st** [comment]
Ugh. I'll go fix the Jenkinsfile.

**Pull request merged**